manifests/init.pp

```diff
@@ -162,11 +163,30 @@
 
   include certs::pulp_client
 
+  file { $foreman_cert_dir:
+    ensure => directory,
+    owner  => $user,
+    group  => $group,
+    mode   => '0775',
+  }
+
+  class { 'certs::foreman':
+    client_cert => "${foreman_cert_dir}/client_cert.pem",
+    client_key  => "${foreman_cert_dir}/client_key.pem",
+    ssl_ca_cert => "${foreman_cert_dir}/proxy_ca.pem",
+    owner       => $user,
+    group       => $group,
+    before      => Class['katello_devel::config'],
+  }
+
+  $ssl_ca_file = "${foreman_cert_dir}/proxy_ca.pem"
+  $ssl_certificate = "${foreman_cert_dir}/client_cert.pem"
+  $ssl_priv_key = "${foreman_cert_dir}/client_key.pem"
```

Comment:
If you define it like this, then I'd say it's better to define it earlier and use the variables as parameters to `certs::foreman`: https://github.com/theforeman/puppet-katello_devel/pull/267